lib/build-env.ts

```diff
@@ -3,15 +3,17 @@ import { aws_codebuild as cbuild } from "monocdk-experiment";
 export interface BuildEnvironmentProps {
   computeType?: cbuild.ComputeType;
   privileged?: boolean;
+  /** @deprecated */
   env?: { [key: string]: string };
+  environment?: { [key: string]: string };
```

Comment:
Why the deprecation? Seems like a similar type...

Reply:
What changes is that the callers pass `environment`, not `env`, because everywhere else this is called `environment`.

Reply:
okay, so basically this is just a rename. the PR title is quite confusing. I would use `feat: rename "env" to "environment" for BuildEnv`